gdal/ogr/ogr_api.cpp

```diff
@@ -32,14 +32,53 @@
 #include "ogr_api.h"
 
 #include <cstddef>
+#include <regex>
 
 #include "cpl_error.h"
 #include "ogr_geometry.h"
+#include "ogr_geos.h"
 
 CPL_CVSID("$Id$")
 
 static bool bNonLinearGeometriesEnabled = true;
 
+/************************************************************************/
+/*                         OGRGetGEOSVersion()                          */
+/************************************************************************/
+
+/** \brief Get the GEOS version
+ *
+ * @param pnMajor Pointer to major version number, or NULL
+ * @param pnMinor Pointer to minor version number, or NULL
+ * @param pnPatch Pointer to patch version number, or NULL
```

Comment:
```suggestion
 * @param pnPatch Pointer to patch version number, or NULL
 * @return TRUE if GDAL is built against GEOS
```
and make changes accordingly